site/category.xml

```diff
@@ -1,155 +1,65 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
-	<description>
-	To install these features, point Eclipse at this site.
-	</description>
+	<description>To install these features, point Eclipse at this site.</description>
 
-	<category-def label="JBoss Tools Base Nightly Build Update Site" name="AllFeatures">

```

Comment:
reduction of number of lines and removal of pointless metadata such as:

    url="features/org.jboss.tools.test.feature.source_0.0.0.jar" version="0.0.0"